docker/docker-compose.yml

```
@@ -0,0 +1,10 @@
+services:
+  frontend:
+    image: ghcr.io/remla23-team11/app:latest
+    ports:
+      - 3000:3000
+    environment:
+      REACT_APP_API_URL: "backend:8080"
+
+  backend:
+    image: ghcr.io/remla23-team11/model-service:latest
```

```suggestion
      REACT_APP_API_URL: "http://localhost:8080"

  backend:
    image: ghcr.io/remla23-team11/model-service:latest
    ports:
      - 8080:8080
```
The API calls are made on the host network.